examples/todoapp/lib/feature_todo_management/blocs/todo_management_bloc.dart

```diff
@@ -108,31 +108,32 @@ class TodoManagementBloc extends $TodoManagementBloc {
           (description, todo) => description ?? todo.description)
       .shareReplay(maxSize: 1);
 
+//  _$setFirstNameEvent
+//             .map(validatorService.validateFirstName)
+//             .startWith(''),
+//         _currentUser.map((user) => user.firstName),
```

Comment:
Seems like some unused code. If left over from testing, just a reminder to remove it.

Reply:
Done